applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -179,25 +179,28 @@ int le_audio_volume_mute(void)
 	return -ENXIO;
 }
 
-int le_audio_play(void)
+int le_audio_play_pause(void)
 {
 	int ret;
 
-	ret = bt_audio_broadcast_source_start(broadcast_source);
-	if (ret) {
-		LOG_WRN("Failed to start broadcast, ret: %d", ret);
+	/* All streams in a broadcast source is in the same state,
+	 * so we can just check the first stream
+	 */
+	if (streams[0].ep == NULL) {
```

Comment:
What if we are synced to 2 streams from 2 different gateways? @gWacey do you have any input here maybe?

Reply:
This is bis_gateway

Reply:
Yeah, my bad